
Tiny compile fix for the raid6 PowerPC/Altivec code on ppc.

  - Rene Rebe <rene@exactcode.de>

--- linux-2.6.11/drivers/md/raid6altivec.uc.vanilla	2005-03-02 16:44:56.407107752 +0100
+++ linux-2.6.11/drivers/md/raid6altivec.uc	2005-03-02 16:45:22.424152560 +0100
@@ -108,7 +108,11 @@
 int raid6_have_altivec(void)
 {
 	/* This assumes either all CPUs have Altivec or none does */
+#ifdef CONFIG_PPC64
 	return cur_cpu_spec->cpu_features & CPU_FTR_ALTIVEC;
+#else
+	return cur_cpu_spec[0]->cpu_features & CPU_FTR_ALTIVEC;
+#endif
 }
 #endif
 
